src/cascadia/TerminalSettingsModel/Profile.h

```diff
@@ -130,7 +127,6 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         GETSET_SETTING(Model::BellStyle, BellStyle, BellStyle::Audible);
 
     private:
-        std::optional<winrt::guid> _Guid{ std::nullopt };
         std::optional<winrt::guid> _ConnectionType{ std::nullopt };

```

Comment:
make this one heritable as well! otherwise ACS won't work for the same reason GUID didn't work.